java/transforms-by-suitability/src/main/java/com/esri/arcgisruntime/sample/transformsbysuitability/DatumTransformationAdapter.java

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2017 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.transformsbysuitability;
+
+import java.util.ArrayList;
+
+import android.content.Context;
+import android.graphics.Typeface;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import com.esri.arcgisruntime.geometry.DatumTransformation;
+import com.esri.arcgisruntime.geometry.GeographicTransformation;
+import com.esri.arcgisruntime.geometry.GeographicTransformationStep;
+
+/**
+ * Adapter that takes an ArrayList of DatumTransformations and shows the name of each transformation, and details
+ * of missing grid files, if any.
+ */
+public class DatumTransformationAdapter extends ArrayAdapter<DatumTransformation> {
+
+  private DatumTransformation defaultValue;
+
+  public DatumTransformationAdapter(Context context, ArrayList<DatumTransformation> transformations) {
+    super(context, 0, transformations);
+  }
+
+  /**
+   * When default transformation is set, the default transformation will be shown in the list with a Bold font.
+   * @param defaultTransformation the default transformation
+   */
+  public void setDefaultTransformation(DatumTransformation defaultTransformation) {
+    defaultValue = defaultTransformation;
+  }
+
+  @Override
+  public View getView(int position, View convertView, ViewGroup parent) {
```

Comment:
Lint is suggesting this be @ NonNull

Reply:
Which parameter? `parent` may be null. `convertView` is specifically checked for null and there's a different code path.